### DISCUSSION TITLE
Unable to send messages to user accounts using an internal domain name that are located at another mail service

### DISCUSSION THREAD

**Discussion by kale1d0code** [Mailu/Mailu#3409] [User support]
I am using Microsoft 365 for emails for a custom domain name
I require the ability for web applications and devices to send email using this domain name
not many devices or web applications support modern email authentication methods so I am looking to use mailu

I want to use mailu to send messages from any arbitrary email address for these web applications and devices into microsoft 365
and also for microsoft 365 to send messages if the user account isn't stored within it as I would also like to have user accounts on this domain

The problem with sending messages with mailu is that none of the messages will send.
I've looked into the logs and it says that it is because the users can't be found in the table.

I'm suspecting that mailu doesn't attempt to send these messages out to microsoft 365.

Is this configuration possible?

I've tried adding the domain as a relayed domain but got an error message stating that the domain name is already in use (I suspect this is because it is used as a mail domain)

**Comment by kale1d0code** [comment]
I've managed to make a lot of progress with this.
I have made a copy of the main.cf file to use as an override file

I've replaced all the instances of the podop variable as this doesn't load correctly when used in the override file
and I made sure a pcre transport map is configured.

The only problem is that mailu keeps rejecting messages intended for the other mail server stating the user doesn't exist in its own virtual user table.
The only way around this I've found so far is to make a copy of the addresses within mailu so that this error message doesn't prevent the transport rules from kicking in.


If there is a way to attempt transport before checking the virtual user table?
it would solve my issue correctly.

**Reply by gonzalo** [reply]
Hi, I'm dealing exactly with the same issue.
We have google workspace accounts on our main domain (.com), but we want to use some email addreses to make some bulk emails,  bypassing the gmail limits. We have been using mailu with other domains (.info, .co) during last years, but now we would like to use mailu as alternative smtp server for marketing campaigns with our main .com domain.

First of all we've set our marketing webtool ip address as relay 

Using authenticated email user does not allow us to send messages with random xxx.com accounts as we receive a message from email server claiming that generic user does not own that account
   553 5.7.1 <toby@xxxxxx.nl>: Sender address rejected: not owned by user no_reply@xxxxxxx.info

Using not authenticated connections over tls results in a:
  Expected response code 250 but got code "530", with message "530 5.7.1 Authentication required              

Using non authenticated connections over 25, results in a 
  4.7.1 Try again later; from=<toby@xxxxxx.nl> to=<gonzalo@xxxxxx.com> proto=ESMTP helo=<mkt.xxxxxxxx.com>

... and message is not sent...

Any clues about how can this be achieved? My DNS spf records are properly set. 

I'm able to send email with xxxxx.com address if I set that domain on mailu, create account and use webmail. But then our mailu server is not able to send emails to our xxxxx.com as it redirects all those emails internally.
